docs/pages/router/advanced/custom-tabs.mdx

```diff
@@ -236,7 +236,11 @@ export const TabButton = forwardRef(
 
 ### Hooks
 
-All components also have a hook version and allows to you create the same functionality without the restrictions of using components. Using hooks, you have complete control over the render tree. See the [Router UI Reference](/versions/latest/sdk/router-ui/) for a full list of the hooks available.
+All components also have a hook version giving you control over the render tree. See the [Router UI Reference](/versions/latest/sdk/router-ui/) for a full list of the hooks available.
+
+Using hooks is considered advanced usage of this library. For most use-cases, using the components with `asChild` should give you enough control over the render tree.
+
+If you are developing a custom `<TabTrigger />`, you may also need to develop a custom `<TabList />` as `<TabList />` uses the [`useTabsWithChildren()`](https://docs.expo.dev/versions/latest/sdk/router-ui/#usetabswithchildrenoptions) which requires using the exported `<TabTrigger />` component
```

Comment:
```suggestion
If you are developing a custom `<TabTrigger />`, you may also need to develop a custom `<TabList />` as `<TabList />` uses the [`useTabsWithChildren()`](/versions/latest/sdk/router-ui/#usetabswithchildrenoptions) which requires using the exported `<TabTrigger />` component.
```

Reply:
`TabList` doesn't call `useTabsWithChildren`, `Tabs` does. And users can't use `useTabsWithChildren` to make their own tabs anyway because it has a hard dependency on `parseTriggersFromChildren` which calls `isTabTrigger` that only checks for the expo version of `TabTrigger` so all these functions would need to be replaced by custom ones. https://github.com/expo/expo/issues/33752

IMO it is better to remove this API entirely or make it actually usable